data/ecosystems/d/drop-wireless.toml

```diff
@@ -0,0 +1,25 @@
+# Ecosystem Level Information
+title = "drop-wireless"
```

Comment:
@GiupiDeLuca, can we use "Drop Wireless" for the tittle instead?

Reply:
Sure, will fix now!